test/unit_test/test_data_root.py

```diff
@@ -53,6 +32,17 @@ def get_dataloader_config():
                 "io_config": {"input_names": ["input"], "output_names": ["output"], "input_shapes": [(1, 1)]},
             },
         },
+        "data_configs": [
+            {
+                "name": "test_data_config",
+                "type": "DummyDataContainer",
+                "load_dataset_config": {
+                    "type": "dummy_dataset",
+                    "params": {"data_dir": "data", "input_shapes": [1], "size": 1},
+                },
+                "dataloader_config": {"params": {"batch_size": 16}},
```

Comment:
the dataloader component is `no_auto_batch_dataloader` so batch size is ignored

Reply:
Fixed.